src/diffusers/models/attention_processor.py

```diff
@@ -1007,15 +1013,19 @@ def __call__(
         if attn.group_norm is not None:
             hidden_states = attn.group_norm(hidden_states.transpose(1, 2)).transpose(1, 2)
 
-        query = attn.to_q(hidden_states, scale=scale)
+        query = attn.to_q(hidden_states, scale=scale) if not USE_PEFT_BACKEND else attn.to_q(hidden_states)
```

Comment:
Same argument as above, just instead of `*args`, use `**kwargs` (though I'm not sure why `scale` is a keyword here but positional above).